ocfweb/stats/daily_graph.py

```diff
@@ -85,6 +85,7 @@ def get_daily_plot(day):
     """Return matplotlib plot representing a day's plot."""
     start, end = get_open_close(day)
     profiles = UtilizationProfile.from_hostnames(list_desktops(public_only=True), start, end).values()
+    desks_count = len(list_desktops(public_only=True))
```

Comment:
Maybe only call `list_desktops` once here and put it into a variable? Every time it is called it makes an external call to LDAP (see [ocflib/lab/stats.py#list_desktops](https://github.com/ocf/ocflib/blob/327dff4a18ba9805aabae8f4c3b4b47f16c42ec5/ocflib/lab/stats.py#L141-L149)), so it seems like a good idea to only make that call once instead of twice.